### PR TITLE
Added locale: zh-CN to Microsoft.SQLServerManagementStudio.yaml

### PR DESCRIPTION
**Pull request by VAllens**
Added locale: zh-CN

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27700)

### CODE REVIEW

**Comment by VAllens** [comment]
It is based on the original format and only add `zh-CN` locale support, no other changes

**Comment by KaranKad** [comment]
For more than one installer you need to use multi-manifest format, take firefox for example [manifests/m/Mozilla/Firefox/92.0](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Mozilla/Firefox/92.0)
https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/27700#issuecomment-919938278

**Comment by VAllens** [comment]
@KaranKad okay, thanks. 👍

File: manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Copyright: Copyright (c) Microsoft Corporation
+ShortDescription: Microsoft.SQLServerManagementStudio
+Description: SQL Server Management Studio (SSMS) is an integrated environment for managing any SQL infrastructure, from SQL Server to Azure SQL Database. SSMS provides tools to configure, monitor, and administer instances of SQL Server and databases. Use SSMS to deploy, monitor, and upgrade the data-tier components used by your applications, and build queries and scripts.
+Tags:
+  - SQL
+  - SQL Server
+  - SQL Management Studio
+  - SSMS
+  - Management Studio
+Moniker: SSMS
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.en-US.yaml:21]: Someone gonna correct this sooner or later
```suggestion
  - sql
  - sql-server
  - sql-management-studio
  - ssms
  - management-studio
Moniker: ssms
```

File: manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    Scope: machine
+    InstallerType: exe
+    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-ENU.exe
+    InstallerSha256: 03E567B7475EC2E2DA237D70E9A833DEABF52CA66D521F340A1EBC8FDEA91170
+  - Architecture: x64
+    InstallerLocale: zh-CN
+    Scope: machine
+    InstallerType: exe
+    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-CHS.exe
+    InstallerSha256: 7266B8E85E2AA843829315DF85D176E5E618B6FA7D2534481D9BB7F2CF0880FB
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.installer.yaml:30]: Corrects the InstallerType, adds /norestart to the Custom field as a temporary fix until they can detect reboot behavior 
```suggestion
InstallerSwitches:
  Custom: /norestart
Installers:
  - Architecture: x64
    InstallerLocale: en-US
    Scope: machine
    InstallerType: burn
    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-ENU.exe
    InstallerSha256: 03E567B7475EC2E2DA237D70E9A833DEABF52CA66D521F340A1EBC8FDEA91170
  - Architecture: x64
    InstallerLocale: zh-CN
    Scope: machine
    InstallerType: burn
    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-CHS.exe
    InstallerSha256: 7266B8E85E2AA843829315DF85D176E5E618B6FA7D2534481D9BB7F2CF0880FB
```

File: manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+PackageVersion: 15.0.18386.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms
+Author: Microsoft
+PackageName: Microsoft SQL Server Management Studio
+PackageUrl: https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms
+License: Copyright (c) Microsoft Corporation
+Copyright: Copyright (c) Microsoft Corporation
+ShortDescription: Microsoft.SQLServerManagementStudio
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.en-US.yaml:13]: Better ShortDescription
```suggestion
ShortDescription: SQL Server Management Studio (SSMS) is an integrated environment for managing any SQL infrastructure
```

File: manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 15.0.18386.0
+PackageLocale: zh-CN
+Publisher: Microsoft Corporation
+PublisherUrl: https://docs.microsoft.com/zh-CN/sql/ssms/download-sql-server-management-studio-ssms
+Author: Microsoft
+PackageName: Microsoft SQL Server Management Studio
+PackageUrl: https://docs.microsoft.com/zh-CN/sql/ssms/download-sql-server-management-studio-ssms
+License: 版权所有 (c) Microsoft Corporation
+Copyright: 版权所有 (c) Microsoft Corporation
+ShortDescription: Microsoft.SQLServerManagementStudio
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.zh-CN.yaml:13]: I have no idea what the Description says, can you add a better ShortDescription?

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.zh-CN.yaml (hunk 4 above)
Comment (VAllens) [reply manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.zh-CN.yaml]: Sure. I just updated the short description. :)

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**